.github/workflows/benchmark-pg_search.yml

```diff
@@ -55,6 +55,22 @@ jobs:
         working-directory: pg_search/
         run: echo "PGRX_VERSION=$(cargo tree --depth 1 -i pgrx -p pg_search | head -n 1 | cut -f2 -dv)" >> $GITHUB_ENV
 
+      - name: Install Rust Cache
+        if: steps.check_skip.outputs.skip_remaining_steps != 'true'
+        uses: ubicloud/rust-cache@v2
+        with:
+          prefix-key: "v1-rust"
+          key: ${{ matrix.pg_version }}-${{ env.PGRX_VERSION }}
+          cache-targets: true
+          cache-all-crates: true
+
+      - name: Cache criterion output
+        if: steps.check_skip.outputs.skip_remaining_steps != 'true'
+        uses: actions/cache@v3
```

Comment:
I think we'll need `ubicloud/cache@v3` here for it to work with Ubicloud

Reply:
actually, doesn't seem like it.  Do you know where ubiclouds docs are for this stuff?